app/views/admin/roles/_form.html.erb

```diff
@@ -5,48 +5,116 @@
 
 <%= form_for role, as: :role, url: role_url_for(role) do |form| %>
   <%= form.errors %>
+    <%= render "govuk_publishing_components/components/input", {
+        label: {
+          text: "Role title"
```

Comment:
text: "Role title (required)"

and also slightly further down we can add the red error text by adding

```
 error_items: errors_for(role.errors, :name)
```